tests/chains.js

```diff
@@ -57,6 +57,8 @@ tape('[Common]: Initialization / Chain params', function (t) {
     st.equal(c.genesis().hash, '0x6341fd3daf94b748c72ced5a5b26028f2474f5f00d824504e4fa37a75767e177', 'rinkeby')
     c.setChain('kovan')
     st.equal(c.genesis().hash, '0xa3c565fc15c7478862d50ccd6561e3c06b24cc509bf388941c25ea985ce32cb9', 'kovan')
+    c.setChain('goerli')
+    st.equal(c.genesis().timestamp, '0x5bdda800', 'goerli')
```

Comment:
Why not check the hash?

Reply:
There was only ``mixHash`` provided in the Goerli [genesis specification(s)](https://github.com/goerli/testnet/blob/master/geth/goerli.genesis) and I wasn't sure how this relates to the generic ``hash`` field here, so I wanted to take a parameter which is disambiguous.

Reply:
The genesis spec formats that Geth and Parity use don't explicitly specify the hash (and state root). It is usually computed by the client, but in our case, we meed to compute it offline and enter it into the spec.